101-acs-swarm/azuredeploy.parameters.json

```diff
@@ -6,7 +6,7 @@
       "value": "GEN-UNIQUE"
     },
     "agentCount": {
-      "value": "1"
+      "value": 1
```

Comment:
I haven't changed anything in the metadata file, so I'm not sure that my change is the issue here ;-)

I've taken a look at the metadata and it looks like there's a newline issue (maybe from pasting content). I've pushed another update that will hopefully fix that.